plutus-core/testlib/PlutusCore/Generators/QuickCheck/Builtin.hs

```diff
@@ -55,44 +56,69 @@ The 'Arbitrary' instances for 'Integer' and 'Int64' only generate small integers
     >>> fmap (any ((> 30) . abs) . concat . concat . concat) . sample' $ arbitrary @[[[Int]]]
     False
 
-We want to at least occasionally generate some larger ones, which is what the 'Arbitrary'
-instance for 'Int64' does:
+We want to generate larger ones, including converted-to-Integer 'minBound' and 'maxBound' of various
```

Comment:
Would it be feasible to get a sensible generator by gluing together the ones for `Int8`, `Int16`, ...?  I've no idea how those behave at the endpoints, so maybe that wouldn't work.

Reply:
Hmmm.  It says above that "The 'Arbitrary' instances for 'Integer' and 'Int64' only generate small integers", but the generators for `Int` and `Int64` seem to be different:
```
ghci> fmap (any ((> 30) . abs) . concat . concat . concat) . sample' $ arbitrary @[[[Int]]]
False
ghci> fmap (any ((> 30) . abs) . concat . concat . concat) . sample' $ arbitrary @[[[Int64]]]
True
```
However
```
ghci> fmap (any ((> 30000) . abs) . concat . concat . concat) . sample' $ arbitrary @[[[Int16]]]
False
```
`maxBound` is 32768, so it looks as if it's not reaching the top.  Trying to use that probably wouldn't be helpful then.

Reply:
> "The 'Arbitrary' instances for 'Integer' and 'Int64' only generate small integers"

That's a typo. Thanks! Should be `'Integer' and 'Int'`, you can see that it's `Integer` and `Int` in the examples right below that line. Fixed the typo.